web-console/knikubevirt/vm-details/devices-and-resources/devices-and-respurces.md

```diff
@@ -0,0 +1,40 @@
+# VM details - Devices-and-resources
+
+### Virtual machine
+
+## From VM details page
+![VM details - Overview tab - default](img/VM-2-0.jpg)
+Within the VM details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.
```

Comment:
Where we have "blank" I wonder if we could have something else to better indicate what the purpose of this section is. It feels like it could be more connected to the "flavor" concept since we are leaning heavily on that for resources in the create flow.

Reply:
How about:
**Scheduling policy**
- Best effort (share resources with other workloads)
- Guaranteed (Dedicate resources for this workload only)

**Resources**
- Share resources with other workloads
- Dedicate resources for this workload only

Reply:
Tomas agreed to:
“Resources”:
(followed by the options)
- Share resources with other workloads
- Dedicate resources for this workload only